vivisect/renderers/__init__.py

```diff
@@ -160,6 +160,12 @@ def renderLocation(self, mcanv, loc, name, isfunc, cmnt, extra):
 
         elif ltype == LOC_STRUCT:
 
+            maxnamelen = 0
+            for soff, sind, sname, sobj in extra.vsGetPrintInfo():
+                snamelen = len(sname)
+                if snamelen > maxnamelen:
+                    maxnamelen = snamelen
```

Comment:
We should give some indication here that we've truncated the name. Maybe like appending a "..."?

Reply:
this doesn't truncate.  it sizes to the largest name so they're all aligned.